manifests/init.pp

```diff
@@ -133,9 +148,12 @@
   Integer[0] $keep_daily                                   = 60,
   Integer[0] $keep_within                                  = 30,
   String[1] $compression                                   = 'lz4',
+  Stdlib::Absolutepath $working_directory                  = '/',
```

Comment:
can you please split this up into multiple PRs? Adding this parameter for example isn't really related to the PR title

Reply:
removed from MR